### ISSUE TITLE
The motion amplitude decreased after training

### ISSUE THREAD

**Issue by Tracyyyyyyyyyy** [Doubiiu/DynamiCrafter#84]
Thank you for your great work. We tried fine-tuning with a learning rate of 1e-5, frame stride set to random in 1-6, resolution: [576, 1024], video_length: 16. After training for a period of time, the motion amplitude of the model output results decreased, and some videos even became still. We did an experiment and adjusted the frame stride to be fixed to 6, but the result was still the same. Can you give us some advice on this training issue? Or what are the parameters for training? Thank u

**Comment by Doubiiu** [comment]
Hi. I am not sure about the reason. You can try to train only part of the model parameters (e.g. keeping the temporal layers frozen), as usually I will keep those temporal layers frozen.

**Comment by SpringtoString** [comment]
> Hi. I am not sure about the reason. You can try to train only part of the model parameters (e.g. keeping the temporal layers frozen), as usually I will keep those temporal layers frozen.



> Hi. I am not sure about the reason. You can try to train only part of the model parameters (e.g. keeping the temporal layers frozen), as usually I will keep those temporal layers frozen.



> Hi. I am not sure about the reason. You can try to train only part of the model parameters (e.g. keeping the temporal layers frozen), as usually I will keep those temporal layers frozen.

I found that the only thing with "temopral" in its name is temopral_conv. Is some transformer_blocks related to temopral? transformer_blocks.number is hard to distinguish.

**Comment by Doubiiu** [comment]
Hi, I hope this function can help.
```
import re
def matches_temporal(s):
    pattern_input_output = r"model\.diffusion_model\.(input|output)_blocks\.[0-9]{1,2}\.2\.(norm|proj_in|transformer_blocks|proj_out).*"
    pattern_middle = r"model\.diffusion_model\.middle_block\.2\.(norm|proj_in|transformer_blocks|proj_out).*"
    return bool(re.match(pattern_input_output, s)) or bool(re.match(pattern_middle, s)) or 'temopral_conv' in s or 'init_attn' in s
```

**Comment by FDoKE** [comment]
I encountered a similar issue but found out that if I generate videos with dimensions similar to the training data, there is more expected movement. If I generate at 1024x576, there is almost nothing happening on the finetuned model, but if I, for example, use 800x480, the finetuned movements start to occur much more
Maybe we should resize dataset to 1024x576?

**Comment by zhuhz22** [comment]
> Thank you for your great work. We tried fine-tuning with a learning rate of 1e-5, frame stride set to random in 1-6, resolution: [576, 1024], video_length: 16. After training for a period of time, the motion magnitude of the model output results decreased, and some videos even became still. We did an experiment and adjusted the frame stride to be fixed to 6, but the result was still the same. Can you give us some advice on this training issue? Or what are the parameters for training? Thank u

Hi, probably this problem can be eased via our recent work: [Identifying and Solving Conditional Image Leakage in Image-to-Video Diffusion Model](https://arxiv.org/abs/2406.15735). You could try our plug-and-play inference strategy on your trained model. The code is [here](https://github.com/thu-ml/cond-image-leakage/tree/main?tab=readme-ov-file).

**Comment by yujiangpu20** [comment]
> Hi, I hope this function can help.
> 
> ```
> import re
> def matches_temporal(s):
>     pattern_input_output = r"model\.diffusion_model\.(input|output)_blocks\.[0-9]{1,2}\.2\.(norm|proj_in|transformer_blocks|proj_out).*"
>     pattern_middle = r"model\.diffusion_model\.middle_block\.2\.(norm|proj_in|transformer_blocks|proj_out).*"
>     return bool(re.match(pattern_input_output, s)) or bool(re.match(pattern_middle, s)) or 'temopral_conv' in s or 'init_attn' in s
> ```

Can I simply freeze all the 'TemporalTransformer' Block in the U-Net? Or some other layers should also be frozen.

**Comment by SpringtoString** [comment]
I think the simplest method to solve slow motion problem is that filtering low score video by calculating optical flow score

**Comment by volcverse** [comment]
> I think the simplest method to solve slow motion problem is that filtering low score video by calculating optical flow score

Hello. I met the same problem that the generated videos remain static. Have you solved the problem? I'm now trying to freeze temporal layer and see what will happen.

**Comment by xlnn** [comment]
> Thank you for your great work. We tried fine-tuning with a learning rate of 1e-5, frame stride set to random in 1-6, resolution: [576, 1024], video_length: 16. After training for a period of time, the motion magnitude of the model output results decreased, and some videos even became still. We did an experiment and adjusted the frame stride to be fixed to 6, but the result was still the same. Can you give us some advice on this training issue? Or what are the parameters for training? Thank u

Hello, how much GPU memory are you using and how many GPUs do you have? I’m using four 32GB GPUs and I'm still getting an out-of-memory error. Could you please share your experience? Thank you!
